src/main/java/com/query/query/mensageria/DisciplinaReceiver.java

```diff
@@ -6,17 +6,19 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import com.query.query.dto.DisciplinaDto;
+import com.projetogerenciamentocurso.gerenciamentocurso.dto.DisciplinaDto;
+import com.projetogerenciamentocurso.gerenciamentocurso.dto.ProfessorDto;
 import com.query.query.exception.DisciplinaException;
 import com.query.query.model.Disciplina;
+import com.query.query.model.Professor;
 import com.query.query.repository.DisciplinaRepository;
 
 @Component
 public class DisciplinaReceiver {
 
 	@Autowired
 	public DisciplinaRepository disciplinaRepository;
-
+	
 	@RabbitListener(queues = "disciplinaCriar")
 	public void inserirDisciplina(DisciplinaDto disciplinaDto) {

```

Comment:
Erro abaixo ocorre ao rodar o teste de criar disciplina no Core e receber o evento no Query.

`Caused by: org.springframework.messaging.converter.MessageConversionException: Cannot convert from [com.projetogerenciamentocurso.gerenciamentocurso.dto.DisciplinaDto] to [com.projetogerenciamentocurso.gerenciamentocurso.dto.ProfessorDto] for GenericMessage [payload=DisciplinaDto(idDisciplina=4, descricao=teste4, cargaHoraria=10, sigla=tes, professor=null), headers={amqp_receivedDeliveryMode=PERSISTENT, amqp_receivedRoutingKey=disciplina_criar, amqp_receivedExchange=amq.direct, amqp_deliveryTag=2, amqp_consumerQueue=professorCriar, amqp_redelivered=false, id=640c0d58-e593-fdde-620e-ab7be61ab83a, amqp_consumerTag=amq.ctag-5-Sf6nDLd0ZGsJj1iQI_nw, amqp_lastInBatch=false, contentType=application/x-java-serialized-object, timestamp=1612907062614}]
	at org.springframework.messaging.handler.annotation.support.PayloadMethodArgumentResolver.resolveArgument(PayloadMethodArgumentResolver.java:145) ~[spring-messaging-5.3.2.jar:5.3.2]
	at org.springframework.messaging.handler.invocation.HandlerMethodArgumentResolverComposite.resolveArgument(HandlerMethodArgumentResolverComposite.java:117) ~[spring-messaging-5.3.2.jar:5.3.2]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.getMethodArgumentValues(InvocableHandlerMethod.java:148) ~[spring-messaging-5.3.2.jar:5.3.2]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:116) ~[spring-messaging-5.3.2.jar:5.3.2]
	at org.springframework.amqp.rabbit.listener.adapter.HandlerAdapter.invoke(HandlerAdapter.java:53) ~[spring-rabbit-2.3.2.jar:2.3.2]
	at org.springframework.amqp.rabbit.listener.adapter.MessagingMessageListenerAdapter.invokeHandler(MessagingMessageListenerAdapter.java:221) ~[spring-rabbit-2.3.2.jar:2.3.2]
	... 14 common frames omitted`